lib/ui/wonder_illustrations/common/wonder_illustration_builder.dart

```diff
@@ -26,7 +26,7 @@ class WonderIllustrationBuilder extends StatefulWidget {
 }
 
 class WonderIllustrationBuilderState extends State<WonderIllustrationBuilder> with SingleTickerProviderStateMixin {
-  late final anim = AnimationController(vsync: this, duration: $styles.times.med * .75)
+  late final anim = AnimationController(vsync: this, duration: $styles.customTime.duration(450))
```

Comment:
Why change this at all? Wouldn't we be able to leave the existing code and it will be 1ms anyways? 

I think anywhere where we were doing a multiple of one of the existing times values, we should just keep it as is.